### PR TITLE
Add basic PHP file writing functions

### PR DESCRIPTION
**Pull request by RichardLim00**
2 different fopen mode, each with different properties and 1 file writing function.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/php/functions/files/file-writing/index.md [hunk 1]
```diff
@@ -8,6 +8,53 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+PHP had prepared a few functions to help user save information outside of PHP to be used later.
```
Comment (co16353sidak) [inline guide/english/php/functions/files/file-writing/index.md]: the line here should be `has prepared`

File: guide/english/php/functions/files/file-writing/index.md [hunk 2]
```diff
@@ -8,6 +8,53 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
+
+## fopen("fileName.txt","w")
+Like most of the file handling functions, before conduct any file operation we must prepare the file for the operation first.
```
Comment (co16353sidak) [inline guide/english/php/functions/files/file-writing/index.md]: line should read `before conducting any`

File: guide/english/php/functions/files/file-writing/index.md [hunk 3]
```diff
@@ -8,6 +8,53 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
...
+<?php
+
+  $fileHandler = fopen("fileName.txt", "w");
+  
+?>
+```
+$fileHandler is the file handling variable, the function to write should pass this variable as parameter.
+
+## fwrite()
+This is the function to add content to your file. This function should contain two variable, the first one should be the file handling variable and the second should be the content you wish to add to the file.
```
Comment (co16353sidak) [inline guide/english/php/functions/files/file-writing/index.md]: `two variable` -> `two variables`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
I have suggested a few changes to some grammatical errors, please make the above changes into the article @RichardLim00

File: guide/english/php/functions/files/file-writing/index.md (hunk 1 above)
Comment (RichardLim00) [reply guide/english/php/functions/files/file-writing/index.md]: Thanks for the suggestion!

**Comment by fadaile** [comment]
language: node_js

node_js:
  - 'lts/*'

cache:
  directories:
    - '$HOME/.npm'

env:
  global:
    - NO_UPDATE_NOTIFIER=1
    - NODE_NO_WARNINGS=1

before_install:
  - npm config set loglevel warn

install: npm ci

before_script:
  - git config --global user.email team+camperbot@freeCodeCamp.org
  - git config --global user.name "CamperBot"

jobs:
  include:
    - stage: Lint javaScript
      script:
        - npm run ensure-env
        - npm run lint

    - stage: Unit and Integration tests
      script: npm test

    - stage: Artifacts builds
      if: branch = master && type != pull_request
      script: bash tools/ci-scripts/trigger-build.sh
      install: skip
      before_script: skip

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**